swagger.yaml

```diff
@@ -304,7 +351,69 @@ paths:
                   $ref: "#/components/schemas/response"      
         400:          
           description: 'failed operation'  
-
+'/articles/{articleId}/comments':
```

Comment:
Good work so far @olorunwalawrence , is the `'/articles/{articleId}/comments':` feature also part of this PR?